benchmarks/device_uvector/device_uvector_bench.cu

```diff
@@ -14,33 +14,39 @@
  * limitations under the License.
  */
 
-#include <benchmark/benchmark.h>
-
-#include <cuda_runtime_api.h>
 #include <rmm/device_uvector.hpp>
 #include <rmm/device_vector.hpp>
 #include <rmm/mr/device/cuda_memory_resource.hpp>
 #include <rmm/mr/device/per_device_resource.hpp>
 #include <rmm/mr/device/pool_memory_resource.hpp>
 
+#include <benchmark/benchmark.h>
+
+#include <cuda_runtime_api.h>
+
 static void BM_UvectorSizeConstruction(benchmark::State& state)
 {
   rmm::mr::cuda_memory_resource cuda_mr{};
   rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource> mr{&cuda_mr};
   rmm::mr::set_current_device_resource(&mr);
 
-  for (auto _ : state) {
+  for (auto _ : state) {  // NOLINT(clang-analyzer-deadcode.DeadStores)
     rmm::device_uvector<int32_t> vec(state.range(0), rmm::cuda_stream_view{});
     cudaDeviceSynchronize();
   }
 
-  state.SetItemsProcessed(state.iterations());
+  state.SetItemsProcessed(static_cast<int64_t>(state.iterations()));
 
   rmm::mr::set_current_device_resource(nullptr);
 }
+
+const auto range_multiplier{10};
+const auto range_start{10'000};
+const auto range_end{1'000'000'000};
+
 BENCHMARK(BM_UvectorSizeConstruction)
-  ->RangeMultiplier(10)
-  ->Range(10'000, 1'000'000'000)
+  ->RangeMultiplier(range_multiplier)
+  ->Range(range_start, range_end)
```

Comment:
I'm all about eliminating magic numbers, but this seems superfluous. The meaning of these values is quite clear from the context of the function.

Reply:
Reverting.